torch/_refs/__init__.py

```diff
@@ -887,14 +887,15 @@ def _ref(
                 "Received a rhs Python scalar to an elementwise binary operation that does not accept rhs scalars!"
             )
 
-        if (
-            not supports_two_python_scalars
-            and isinstance(a, Number)
-            and isinstance(b, Number)
-        ):
-            raise ValueError(
-                f"Receive two Number inputs to an elementwise binary operation {prim}!"
-            )
+        # TODO: enable this for operations that support it, like add
```

Comment:
This has been supported in https://github.com/pytorch/pytorch/pull/87759 
Changes here no longer need to be merged to master.